### DISCUSSION TITLE
Different speed when editting mapping

### DISCUSSION THREAD

**Discussion by CamelSounds** [helgoboss/helgobox#1204] [General]
First of thanks for this, it's awesome software :)

I'm using the midi fighter twister with velocity sensitive on every encoder. All are set to endless in MF Utility.
I'm running into an issue and I'm not sure where it originates from. When editting a mapping in the main compartment, I set the step sizes to what feels right. But when I close the editor it seems sluggish. Where half a turn would be enough for a full rotation now it takes a whole turn.

I set the max step size "high enough" at 100 in both the controller mapping and the main compartment mapping as you can see in the screenshots. I've used the controller mapping in Reapack. 

![image](https://github.com/user-attachments/assets/f71de237-6f4a-478f-bd8d-e7a0fa9ce7cb)
![image](https://github.com/user-attachments/assets/aaeafefa-0286-4a96-8984-34a315a29b5e)

I hope you can help me!

**Comment by CamelSounds** [comment]
I thought it was fixed but no issue remains :(

**Comment by helgoboss** [comment]
Hmm. Maybe you can increase the minimum step size?

I assume you followed all the Twister-related instructions in the "Controllers" guide?

**Comment by CamelSounds** [comment]
Yess, I did. And yeah I did try a lot of tweaking with the step size. I
might just do a clean install of realearn :)

**Comment by helgoboss** [comment]
No, please don't ;) Reinstalling wouldn't change anything.

If you change the minimum step size, it should definitely have the effect that you need to turn the encoder less. That should at least provide a workaround. That works, right? The minimum step size is the most important one. The maximum step size is only relevant for limiting the effect of acceleration.

I would be surprised if an open mapping panel influences the effect of encoder turning but I also can't rule it out. No way to test it now as I'm AFK until next Monday.

**Comment by CamelSounds** [comment]
Yeah, I can get it in the range where I like it, but the setup is a bit of
a pain right now since the edit window seems to have a different
sensitivity. Requires more back and forth but not the biggest issue. Thanks
for getting back to me so soon at least :)

Cheers!

**Comment by helgoboss** [comment]
Which FX are you controlling? I couldn't reproduce this until now. I would like to try it with the same plug-in and parameter that you use.

**Comment by CamelSounds** [comment]
I was using devil-loc by Soundtoys. I believe it was with every fx but not
entirely sure. Thanks for getting back to me! I could perhaps send my
realearn instance? Not sure if that is easily possible.

**Reply by helgoboss** [reply]
The best thing would be if you could send me your project file. You can also send it to info@helgoboss.org.

**Comment by CamelSounds** [comment]
Okay I will! Start of next week I'll have more time so I'll send it then :)
